### PR TITLE
Custom library

### PR DESCRIPTION
**Pull request by vinhliem**
Initial commit

### CODE REVIEW

**Comment by vinhliem** [comment]
Some sites wouldn't detect username is found or not at the moment, still updating...
```
"G2G": {
    "errorType": "response_url",
    "errorUrl": "https://www.g2g.com/{}",
    "regexCheck": "^[A-Za-z][A-Za-z0-9_]{2,11}$",
    "url": "https://www.g2g.com/{}",
    "urlMain": "https://www.g2g.com/",
    "username_claimed": "user"
  },
  "Virgool": {
    "errorMsg": "\u06f4\u06f0\u06f4",
    "errorType": "message",
    "url": "https://virgool.io/@{}",
    "urlMain": "https://virgool.io/",
    "username_claimed": "blue"
  },
  "ebio.gg": {
    "errorType": "message",
    "errorMsg": "hasn't been claimed yet!",
    "request_method": "GET",
    "url": "https://ebio.gg/{}",
    "urlMain": "https:/ebio.gg",
    "username_claimed": "dev"
  },
  "Whonix Forum": {
    "errorType": "status_code",
    "url": "https://forums.whonix.org/u/{}/summary",
    "urlMain": "https://forums.whonix.org/",
    "username_claimed": "red"
  }
```

**Comment by vinhliem** [comment]
I think it good enough for use at the moment. Can you take a look of this @richiemann before merging and using on our provider

File: removed_sites.json
Comment (richiemann) [inline removed_sites.json:1]: Why is this file located in the top level directory?

Comment (richiemann) [reply removed_sites.json:1]: Aside from that I think we can merge this. However I will need the merge first on the searchlight provider, so it goes to Infinia DEV

Comment (vinhliem) [reply removed_sites.json:1]: It's been located by the author which a [description](https://github.com/normadnet/sherlock/blob/master/removed_sites.md) about that. I added more to here and also removing those out of our main json sites

Comment (richiemann) [reply removed_sites.json:1]: Yes, I see, but I don't want that file on that directory (/) - I would prefer for it to be inside /sherlock/resources like data.json

Comment (richiemann) [reply removed_sites.json:1]: We can do this on Monday - please enjoy the weekend - I am just jetlagged  :)

Comment (vinhliem) [reply removed_sites.json:1]: I will change it and push to a dev tomorrow. Please take time to rest 😁

**Comment by vinhliem** [comment]
The search works fine with their user-agent, so then I revert the use of random-user-agent

Screenshot from 2023-12-16 23-25-36
Screenshot from 2023-12-16 23-23-48

**Review by richiemann — APPROVED** [review]

**Pull request merged**